website/theme.config.tsx

```diff
@@ -0,0 +1,232 @@
+              <path
+                d="m64.8833 1.81335-2.8464 2.84638C47.1274 19.5692 22.9543 19.5692 8.04485 4.65972L5.19848 1.81335c-.93479-.93478-2.45037-.93478-3.38515 0-.93479.93478-.93478 2.45037 0 3.38515L4.6597 8.04487c14.9095 14.90953 14.9095 39.08263 0 53.99213l-2.84637 2.8463c-.93479.9348-.93479 2.4504 0 3.3852.93478.9348 2.45037.9348 3.38515 0l2.84637-2.8464c14.90955-14.9095 39.08255-14.9095 53.99205 0l2.8464 2.8464c.9348.9348 2.4504.9348 3.3852 0 .9347-.9348.9347-2.4504 0-3.3852l-2.8464-2.8463c-14.9095-14.9095-14.9095-39.0826 0-53.99213l2.8464-2.84637c.9347-.93478.9347-2.45037 0-3.38515-.9348-.93478-2.4504-.93478-3.3852 0Z"
+                stroke="currentColor"
+                strokeWidth="2"
+              />
+            </svg>
+          </a>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export default {
+  backgroundColor: {
+    dark: '27,27,27',
+  },
+  head: function useHead() {
+    const { frontMatter, title: pageTitle } = useConfig();
+    const { asPath } = useRouter();
+
+    const title = `${pageTitle}${asPath === '/' ? '' : ' | GraphQL'}`;
+    const { description, canonical, image } = frontMatter;
+    return (
+      <>
+        <title>{title}</title>
+        <meta property="og:title" content={title} />
+        {description && (
+          <>
+            <meta name="description" content={description} />
+            <meta property="og:description" content={description} />
+          </>
+        )}
+        {canonical && <link rel="canonical" href={canonical} />}
+        {image && <meta name="og:image" content={image} />}
+        <meta property="og:image" content="/img/og-image.png" />
+        <meta property="twitter:site" content="@graphql" />
+      </>
+    );
+  },
+  banner: {
+    content: (
+      <>
+        🎬 That's a Wrap for GraphQLConf 2024! • Watch the Videos •{' '}
+        <NextLink
+          href="/conf/2024"
+          className="underline after:content-['_→'] after:font-sans"
+        >
+          Check out the recorded talks and workshops
+        </NextLink>
+      </>
+    ),
+    key: 'graphqlconf-2024',
+  },
+  logo: graphQLLogo,
+  docsRepositoryBase:
+    'https://github.com/graphql/graphql.github.io/tree/source',
```

Comment:
```suggestion
    'https://github.com/graphql/graphql-js/tree/16.x.x/website',
```

or

```suggestion
    'https://github.com/graphql/graphql-js/tree/main/website',
```

so Edit On GitHub link will point to right file